Dockerfile

```diff
@@ -12,7 +12,7 @@ ADD . .
 
 # Because of the GoRook deployment architecture, the go.mod is pointing to the stub version and the CI is responsible for replacing it with the real version
 # We download the GoRook explicitly so it would register it in the go.sum
-RUN go get -d github.com/Rookout/GoSDK@v0.1.8
+RUN go get github.com/Rookout/GoSDK@v0.1.9
```

Comment:
Why did you remove the `-d` flag?

Reply:
> Why did you remove the `-d` flag?

It is deprecated and used by default in go 1.18